ang/test/civicase/case/list/directives/sticky-table-header.directive.spec.js

```diff
@@ -104,27 +104,55 @@ describe('civicaseStickyTableHeader directive', function () {
   });
 
   describe('if loading is complete and case is not focussed', function () {
-    beforeEach(inject(function ($timeout) {
-      scope.isLoading = false;
-      scope.caseIsFocused = false;
-      scope.$digest();
-      $timeout.flush(); // Flushing any timeouts used.
-    }));
+    describe('and not viewing the case', function () {
+      beforeEach(inject(function ($timeout) {
+        scope.isLoading = false;
+        scope.caseIsFocused = false;
+        scope.viewingCase = false;
+        scope.$digest();
+        $timeout.flush(); // Flushing any timeouts used.
+      }));
 
-    it('adds min-width style to the table elements', function () {
-      expect(element.find('thead').html()).toContain('style="min-width');
-    });
+      it('adds min-width style to the table elements', function () {
+        expect(element.find('thead').html()).toContain('style="min-width');
```

Comment:
This test is a bit off. Which element is going to have the `min-width` style set? What will its value be? What would happen if another element gets a `min-width` style?

Reply:
Fixed.